src/client/pages/AssessmentSection/DataTable/Table/Row/RowData/Cell/props.ts

```diff
@@ -8,7 +10,9 @@ export type PropsCell = {
   rowIndex: number
   col: Col
   row: Row
+  nodeValue: NodeValue
   datum: string
```

Comment:
these two are the same. we should use only one

Reply:
replace with nodeValue